src/Components/Patient/PatientInfoCard.tsx

```diff
@@ -113,7 +114,11 @@ export default function PatientInfoCard(props: {
 
     return false;
   };
-
+  const username =
+    consultation?.treating_physician_object?.username || "bhuvan";
+  const { data: skills } = useQuery(routes.userListSkill, {
+    pathParams: { username: username },
+  });
```

Comment:
@Sulochan-khadka It is not a good idea to hardcode random usernames into the codebase (even as a fallback). If it was done to handle cases where the object is null, you could make use of the `prefetch` option.

```suggestion
  const { data: skills } = useQuery(routes.userListSkill, {
    pathParams: { username: consultation?.treating_physician_object?.username },
    prefetch: !!consultation?.treating_physician_object?.username
  });
```

Reply:
I apologise for this... Actually I was have some null checks and so had to write some hard coded names... Just forgot to remove them before creating a PR.

Reply:
using username directly will throw an error since it can be undefined. Therefore added nullish coalescing in the latter commit.